modules/cert-manager-acme-http01.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-http01_{context}"]
+// = Using the HTTP01 Ingress solver
+
+= Issuing certificates by using the HTTP01 challenge
+// TODO: Name
+
+TODO: intro
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* TODO
+// * User workload to expose
+// TODO: ^ from gdoc, what does this mean?
```

Comment:
@thejasn @swghosh This was pulled from the google doc as a prereq. Can you explain more specifically what this means? Or if there is a test workload/whatever objects we could add to the procedure to show this?

Reply:
I've updated the statement to be more specific.

Reply:
Service would be part of the user's workload (basically up to the user). Any workload the user wishes to securely expose would require a service object.

Reply:
Okay thanks. I updated this, it's still pretty generic, but I'll leave it as is for now then.